src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "scss/variables/_colors";
+@import "scss/variables/_mixins";
+* {
+  font-family: "Roboto";
+}
+
+.container {
+  @include flex(space-around, center, column);
+  border-radius: 10px;
+  text-align: center;
+  display: inline-flex;
```

Comment:
the `flex` mixin already has the display... either remove the mixin or this `display` declaration

Reply:
done